### PR TITLE
Updated docs with local gem instructions

### PR DESCRIPTION
**Pull request by msecret**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,10 @@ the source files.
...
-2. Run the watching build task in the *cg-style* repository: `npm run watch`
-3. Run `npm install` in the *cg-landing* repository.
-4. Run `npm link` in *cg-landing*.
-5. Run the watching build task in *cg-landing* repository: `npm run watch`
-6. Edit code in the *cg-style* directory and they will propagate down to *cg-landing*
+2. Follow the instructions on the [cg-style readme](https://github.com/18F/cg-style#install-and-use) to install and build the library.
+3. Run the watching build task in the *cg-style* repository: `npm run watch`
+4. Replace the `gem 'cloudgov-style'` line in the `Gemfile` with `'cloudgov-style', :github => '18F/cg-style-gem', :branch => 'master'`.
+6. Run `bundle` again.
+7. Run `bundle config local.cloudgov-style ~/path/to/cg-style/repo` wih the path to where you cloned the cg-style repo.
```
Comment (afeld) [inline README.md]: Maybe easier to tell them to use the `:path` option?

http://bundler.io/gemfile.html

Comment (msecret) [inline README.md]: So in that case, we'd just tell them to add `, :path => './vendor/extracted_library'` rather then change the whole line? That makes more sense

Comment (afeld) [inline README.md]: Yep, that would allow you to change the "Replace the `gem 'cloudgov-style'`" line, and cut out the `bundle config` step.

**Comment by jeremiak** [comment]
@afeld this look good now?

**Comment by afeld** [comment]
Build failing due to https://github.com/wercker/step-bundle-install/issues/14.

**Pull request merged**